backend/coins/btc/account.go

```diff
@@ -476,26 +475,72 @@ func (account *Account) Notifier() accounts.Notifier {
 	return account.notifier
 }
 
-func (account *Account) updateFeeTargets() {
-	defer account.feeTargetsLock.Lock()()
+// feeTargets fetches the available fees. For mainnet BTC it uses mempool.space estimation.
+//
+// For the other coins or in case mempool.space is not available it fallbacks on Bitcoin Core.
+// The minimum relay fee is used as a last resource fallback in case also Bitcoin Core is
+// unavailable.
+func (account *Account) feeTargets() []*FeeTarget {
+	// for mainnet BTC we fetch mempool.space fees, as they should be more reliable.
+	mempoolFees := &accounts.MempoolSpaceFees{}
+	if account.coin.Code() == coin.CodeBTC {
+		_, err := util.APIGet(account.httpClient, mempoolSpaceMirror, "", 1000, mempoolFees)
+		if err != nil {
+			mempoolFees = nil
+			account.log.WithError(err).Error("Fecthing fees from " + mempoolSpaceMirror + " failed")
```

Comment:
fyi there is also `Errorf`: `.Errorf("Fetching fees from %s failed", mempoolSpaceMirror)`

also: type in `Fetching`